src/objects/avatar/Avatar.ts

```diff
@@ -606,6 +607,14 @@ export class Avatar extends RoomObject implements IMoveable, IScreenPositioned {
   private _stopWalking() {
```

Comment:
If you take a look here, unintuitively the `_stopWalking` is also called when a `move` movement is started. This new callback will get triggered when `move` is triggered, even though there is no active walking animation, which doesn't seem right.

https://github.com/jankuss/shroom/blob/3c334982f37300ee291f4d970344950b5e1ebfa4/src/objects/avatar/Avatar.ts#L423

For a solution, see my below comment